rules/windows/c2_grey_list_beacon_sequence.toml

```
@@ -0,0 +1,88 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies network activity from unexpected system applications. This may indicate adversarial activity as these
+applications are often leveraged by adversaries to execute code and evade detection.
+"""
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Unusual Process Network Connection: Sequence"
+risk_score = 21
+rule_id = "1fe3b299-fbb5-4657-a937-1d746f2c711a"
+severity = "medium"
+tags = ["from-eql-rules"]
+type = "eql"
+
+query = '''
+sequence by process.entity_id, process.pid, process.name with maxspan=5m
```

We can discuss best practice and setting precedence of choosing sequence fields which most contribute to performance and add to readability.

---

Let's go with this. We can always add more later if we find some other optimization along the way.

```suggestion
sequence by process.entity_id with maxspan=5m
```

---

wfm - will update the rest for conformity